app/livechat/server/sendMessageBySMS.js

```diff
@@ -36,6 +36,11 @@ callbacks.add('afterSaveMessage', function(message, room) {
 		extraData = Object.assign({}, { rid, userId, fileUpload });
 	}
 
+	if (message.customFields && message.customFields.imageCardURL) {
```

Comment:
@AlexanderKanakis Functionality working great! I was able to get pdf and image as well.

IMO it's better to rename `imageCardURL` to something more generic, as functionality is not limited to Images 😄

Reply:
maybe just 'cardURL'?

Reply:
mediaCardUrl